Modrinth/overrides/config/capes.json5

```diff
@@ -0,0 +1,8 @@
+{ 
```

Comment:
I would prefer having configs symlinked to one location (CF) in the repo to prevent duplicates.

Reply:
Think I managed to do that in 338e3c3377172d3528c5c6df1b6b894b673f407c, never used symlinks in git before though, so I'm not sure :sweat_smile: